pkg/store/bucket.go

```diff
@@ -1030,6 +1032,19 @@ func (s *bucketBlockSet) getFor(mint, maxt, minResolution int64) (bs []*bucketBl
 	if len(bs) == 0 {
 		return s.getFor(mint, maxt, s.resolutions[i])
 	}
+
+	until := len(bs) - 1
+	for bsi := 0; bsi < until; bsi++ {
```

Comment:
`bsi`?

Reply:
`bsi` > `j`? imo easier to recognize that it's just a loop counter rather than a cryptic TLA ;)

Reply:
I agree, I don't remember why I chose such name. I think I had some kind of different version before and in that version I had a variable named `j` already so I chose a different one. Fixed now :smile: